spec/models/like_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  describe 'associations' do
+    it { should belong_to(:user).class_name('User') }
+    it { should belong_to(:post) }
+  end
+
+  describe 'callbacks' do
+    describe 'after_save' do
+      let(:user) { User.create(name: 'Juwairiyya Sadiq') }
+      let(:post) { Post.create(title: 'Test Post') }
+    end
+  end
+end
```

Comment:
- [ ] You forgot to write unit tests for the `increment_likes_counter` method you created in the _**app/models/like.rb**_ file. Kindly write unit tests for this as it ensures that the method works properly. 👍 🖱️ 

#### Hint: You can create some `Like` instances and ensure that the value of the `likes_counter` is equal to the number of likes created. 🔹